mkdocs.yml

```diff
@@ -27,6 +27,7 @@ theme:
 nav:
   - Overview: README.md
   - OpenStack:
+    - contributing/development_environment.md
```

Comment:
actually, we should remove this file, mkdocs isn't used anymore

Reply:
+1, not a blocker for this patch

Reply:
https://github.com/openstack-k8s-operators/data-plane-adoption/pull/272